src/plugins/intel_cpu/src/emitters/x64/cpu_generator.cpp

```diff
@@ -52,6 +52,15 @@ namespace ov {
     } \
 }
 
+#define CREATE_UNDEFINED_EMITTER(node_type) { \
+    [](const snippets::lowered::ExpressionPtr& expr) -> std::shared_ptr<snippets::Emitter> { \
+        return nullptr; \
+    }, \
+    [](const std::shared_ptr<ov::Node>& n) -> std::set<std::vector<element::Type>> { \
+        return node_type::get_supported_precisions(n); \
+    } \
+}
```

Comment:
Change the macros: it shouldn't use static method of a node: target machine can report supported precisions for such operations